src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
You can use same font-family for both @font-faces but different font-weight. Then changing only the font-weight on your elements will do the job.